git_machete/docs.py

```diff
@@ -728,128 +848,146 @@
+
+        Note
+        To make it easier to select branch name from the status output on certain terminals
+        (e.g. Alacritty), you can add an extra
+        space between └─ and branch name by setting git config machete.status.extraSpaceBeforeBranchName true.
+        For example, by default the status is displayed as:
+
+        develop
+        │
+        ├─feature_branch1
+        │
+        └─feature_branch2
+
+        With machete.status.extraSpaceBeforeBranchName config set to true:
+
+        develop
+        │
+        ├─ feature_branch1
+        │
+        └─ feature_branch2
+
+
+   """,
     "traverse": """
-        <b>Usage: git machete t[raverse] [-F|--fetch] [-l|--list-commits] [-M|--merge]
-                                         [-n|--no-edit-merge|--no-interactive-rebase] [--no-detect-squash-merges]
-                                         [--[no-]push] [--[no-]push-untracked]
-                                         [--return-to=WHERE] [--start-from=WHERE] [-w|--whole] [-W] [-y|--yes]</b>
+        <b>Usage:</b><b>
+          git machete t[raverse] [-F|--fetch] [-l|--list-commits] [-M|--merge]
+                                 [-n|--no-edit-merge|--no-interactive-rebase] [--no-detect-squash-merges]
+                                 [--[no-]push] [--[no-]push-untracked]
+                                 [--return-to=WHERE] [--start-from=WHERE] [-w|--whole] [-W] [-y|--yes]
+        </b>
 
         Traverses the branch tree in pre-order (i.e. simply in the order as they occur in the definition file).
         By default `traverse` starts from the current branch.
         This behaviour can, however, be customized using options: `--start-from=`, `--whole` or `-w`, `-W`.
 
         For each branch, the command:
-        * detects if the branch is merged (<b><dim>grey</dim></b> edge) to its parent (aka upstream)
-          - by commit equivalency (default), or by strict detection of merge commits (if `--no-detect-squash-merges` passed),
-          - if so, asks the user whether to slide out the branch from the dependency tree (typically branches are no longer needed after they're merged);
-        * otherwise, if the branch has a <b><red>red</red></b> or <b><yellow>yellow</yellow></b> edge to its parent/upstream (see `status`):
-          - asks the user whether to rebase (default) or merge (if `--merge` passed) the branch onto into its upstream branch - equivalent to `git machete update` with no `--fork-point` option passed;
-
+        * detects if the branch is merged (grey edge) to its parent (aka upstream):
+           - by commit equivalency (default), or by strict detection of merge commits (if `--no-detect-squash-merges` passed),
+           - if so, asks the user whether to slide out the branch from the dependency tree (typically branches are no longer needed after they're merged);
+        * otherwise, if the branch has a [31mred[0m or [33myellow[0m edge to its parent/upstream (see status):
```

Comment:
Whoops

Reply:
I mean this was to be expected since there were no colors in the cmd line docs before and we decided to add them

Reply:
Oh! okay, makes sense, I see